sklearn/decomposition/pca.py

```diff
@@ -202,6 +202,11 @@ class PCA(_BasePCA):
         If ``n_components`` is not set then all components are stored and the
         sum of explained variances is equal to 1.0.
 
+    singular_values_ : array, [n_components]
```

Comment:
hm here all the docstrings seem to be using a different convention. I like ``shape + tuple`` better, that's the standard numpy way. But I don't mind that much.